node/actors/bft/src/tests/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod integration_tests;
+mod unit_tests;
```

Comment:
Bikeshedding: I'd shorten module names to `integration` (or `network`) and `unit` (or something like `dispatch`) since they are already have the enclosing `tests` module.

Reply:
Fixed.